code/modules/mob/living/simple_animal/hostile/necro.dm

```diff
@@ -576,3 +576,54 @@
 				set_light(1, 2, "#5dca31")
 
 #undef RAD_COST
+
+
+/////////////////////////////////////////////////////// 
+///////////////// HEADCRAB ZOMBIES //////////////////// 
+///////////////////////////////////////////////////////
+
+/mob/living/simple_animal/hostile/necro/zombie/headcrab //Not very useful
```

Comment:
because they're a child of zombies there's a possibility that they inherit zombie behavior including eating meat, and player zombies can turn into other kinds of zombies. Investigate if this holds true for headcrab zombies

Reply:
Nevermind, they got can_evolve set to false. Look into...